helm/flux-app/templates/vpa.yaml

```diff
@@ -11,6 +11,8 @@ spec:
     containerPolicies:
     - containerName: manager
       controlledValues: RequestsAndLimits
+      maxAllowed:
+        memory: {{ .Values.resources.helmController.limits.memory }}
```

Comment:
In general I think it's best to let VPA do its thing. So only setting the max allowed mem limit.